src/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj

```diff
@@ -15,12 +15,11 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netstandard-Unix-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netfx-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netfx-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
You didn't miss anything in the docs, we have minimal docs on the subject right now https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#facades with a TODO to talk about how to configure them.

What happens when you set IsPartialFacadeAssembly to true we do a post build process that will find the reference assembly and do a diff of all the types in the ref vs what is in the src library and for any types not in the src library it will inject a type-forward for them to where they exist in the reference libraries. This diff includes any type-forwards in the reference assembly as well which essentially get copied to the src library.